langchain/law_vectorstore.py

```
@@ -0,0 +1,496 @@
+                    if filename not in processed_states:
+                        processed_states[filename] = {}
+                    processed_states[filename]["vectorstore_processed"] = True
+                    processed_states[filename]["document_count"] = len(
+                        documents)
+                    processed_states[filename]["chunk_count"] = len(
+                        split_documents)
+
+                    # 상태 저장
+                    self._save_processed_states(processed_states)
+
+            except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)